core/src/main/java/org/owasp/encoder/Encode.java

```diff
@@ -243,7 +243,7 @@ public static void forHtmlContent(Writer out, String input)
      *
      * <b>Example JSP Usage</b>
      * <pre>
-     *     &lt;div&gt;&lt;%=Encode.forHtmlAttribute(unsafeData)%&gt;&lt;/div&gt;
+     *     &lt;input value=&quot;&lt;%=Encode.forHtml(unsafeData)%&gt;&quot; title=&#39;&lt;%=Encode.forHtml(moreUnsafeData)%&gt;&#39; /&gt;
      * </pre>
```

Comment:
Inspired by the JSP example in `forHtml()` JavaDocs, but demonstrates usage with both single-quotes and double-quotes.